.golangci.yml

```diff
@@ -57,6 +57,8 @@ linters-settings:
       - whyNoLint # checked by nolintlint linter
       - hugeParam # TODO(vtopc): configure(80 bytes is probably not so much) and enable.
       - rangeValCopy # TODO(vtopc): configure(disable for tests) and enable.
+      - appendAssign
+      - commentedOutCode
```

Comment:
https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
It was complaining about code examples in comments and there were some commented out debug logging things.  Seemed like an unnecessary rule.